### PR TITLE
Use inclusive language

### PR DESCRIPTION
**Pull request by gjvoosten**
Use inclusive language in our code where possible.

#### User changes
- none

#### Super User changes
- none

#### Admin changes
- none

#### System admin changes
- [ ] anet.yml or anet-dictionary.yml needs change
- [ ] db needs migration
- [x] documentation has changed
- [ ] graphql schema has changed

### Checklist
  - [x] Described the user behavior in PR body
  - [ ] Referenced/updated all related issues
  - [x] commits follow a `repo#issue: Title` title format and [these 7 rules](https://chris.beams.io/posts/git-commit/)
  - [x] commits have a [clean history](https://epage.github.io/dev/commits/), otherwise PR may be squash-merged
  - [ ] Added and/or updated unit tests
  - [ ] Added and/or updated e2e tests
  - [ ] Added and/or updated data migrations
  - [x] Updated documentation
  - [x] Resolved all build errors and warnings
  - [ ] Opened debt issues for anything not resolved here

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 
-You can submit contributions anonymously or under a pseudonym if you'd like, but we need to be able to reach you at the email address you list when you agree to the DCO. 
+You can submit contributions anonymously or under a pseudonym if you'd like, but we need to be able to reach you at the email address you list when you agree to the DCO.
 
-It probably goes without saying, but contributions you make to this public Department of Defense repository are completely voluntary. When you submit a pull request, you're offering your contribution without expectation of payment and you expressly waive any future pay claims against the U.S. Federal government related to your contribution. 
+It probably goes without saying, but contributions you make to this public Department of Defense repository are completely voluntary. When you submit a pull request, you're offering your contribution without expectation of payment and you expressly waive any future pay claims against the U.S. Federal government related to your contribution.
```
Comment (jf2020) [inline CONTRIBUTING.md:27]: Department of defense was the old name

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 
 ### Open a Pull Request
 
-Once your branch is ready to be merged please open a pull request and assign it to one of the repository admins.  (Currently @hunterp and @nickjs).  
+Once your branch is ready to be merged please open a pull request and assign it to one of the repository admins.
```
Comment (cemalettin-simsoft) [inline CONTRIBUTING.md:31]: Who are the current admins :thinking: ? Might as well write that down for future contributors.

Comment (gjvoosten) [reply CONTRIBUTING.md:31]: Non-authorised users can't assign their PR's anyway (afaik), so I'll just remove that bit.

File: CONTRIBUTING.md (hunk 1 above)
Comment (gjvoosten) [reply CONTRIBUTING.md:27]: Unrelated to this PR.

**Pull request merged**